recipes/libplacebo/recipe.yaml

```diff
@@ -0,0 +1,58 @@
+context:
+  name: libplacebo
+  version: 7.349.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://code.videolan.org/videolan/libplacebo/-/archive/v${{ version }}/libplacebo-v${{ version }}.tar.gz
+  sha256: 79120e685a1836344b51b13b6a5661622486a84e4d4a35f6c8d01679a20fbc86
+
+build:
+  number: 0
+  skip: win
```

Comment:
This is a shared library package, so it needs run_exports pinning the appropriate ABI for downstream packages.

Reply:
Done.